### PR TITLE
 New line on housing protection legislation popups #26 solution 

### PR DESCRIPTION
**Pull request by isthewhiz**
Added a <p> tag that says "If your local city or county protections have expired you may still be protected under state or national legislation." to the enddate legis

### CODE REVIEW

File: src/components/InfoWindow.js [hunk 1]
```diff
@@ -151,6 +151,9 @@ export default props => {
                 <strong>{t('infowindow.policy.protections-end-label')}</strong>{' '}
                 {endDateLegist}
               </p>
+              <p><strong>{t('')}</strong>{' '}
+                {"If your local city or country protections have expired you may still be protected under state or national legislation."}
```
Comment (c5inco) [inline src/components/InfoWindow.js:155]: I feel like this string should be added to the localization set (`src/locale`) if it's a consistent message included with locations that have legislative end dates?

**Review by c5inco — COMMENTED** [review]
Thanks for adding me! Could you provide a test location where I can specifically see this info show in the popup?

File: src/components/InfoWindow.js (hunk 1 above)
Comment (isthewhiz) [reply src/components/InfoWindow.js:155]: I agree but I was having a hard time, when I added this string to that en.json file in the src/locale accessing wasn't working like I thought. There is a lot more going on than slipping in a text line, LOL. Do you think in the src/ info windows I can create a "tag" variable like extendedinfo then use that to call the locale file and just add the text in English, Spanish and German. Would you be able to do a video call sometime when your free. I don't think I'm property explaining the situation?

Comment (c5inco) [reply src/components/InfoWindow.js:155]: Ah I see. Yes I'm free for a call after 5pm PST on Thursday if that works?

**Comment by isthewhiz** [comment]
Thursday works! I’ll reach out then.

**Comment by isthewhiz** [comment]
Hi Chris, I added the line of text in the three files and also added a p tag in the publicinfo pop up so that it appears at the end of the paragraph. lastly i deleted the extra info word because i added it. If you could check it out that would be great. Thanks

File: src/components/InfoWindow.js [hunk 2]
```diff
@@ -192,6 +190,10 @@ export default props => {
                 <strong>{t('infowindow.policy.summary-label')}</strong>{' '}
                 {policy_summary}
               </p>
+              <p></p>
```
Comment (c5inco) [inline src/components/InfoWindow.js:192]: I think you can review this extra `<p>` element

**Review by c5inco — CHANGES_REQUESTED** [review]
Thanks for the changes Isnardo! I think you still need translations for Italian (`it.json`) and Brazilian Portuguese (`pt-BR.json`)?

Also looks like there are some merge conflicts. I'd suggest rebasing your changes on top of master since it looks like the last time you synced your repository was about 8 weeks ago.

**Comment by isthewhiz** [comment]
Thanks for the feed back, I made the changes and updates. I have to correct some typos but will push this up by the weekend

**Comment by c5inco** [comment]
Cool - did you see the change request to remove the extra `<p>` tag?

**Comment by isthewhiz** [comment]
No I’ll remove it, lol

**Comment by isthewhiz** [comment]
I removed it. I missed that line description.

**Review by c5inco — APPROVED** [review]

**Comment by c5inco** [comment]
Great! I think you have some merge conflicts to resolve first though before the PR can be merged.

**Comment by isthewhiz** [comment]
Awesome!! Thanks

**Pull request closed without merging**